pom.xml

```diff
@@ -95,131 +95,186 @@
+        <version.antrun.plugin>1.8</version.antrun.plugin>
+        <version.assembly.plugin>3.1.0</version.assembly.plugin>
+        <version.buildhelper.plugin>3.0.0</version.buildhelper.plugin>
+        <version.bundle.plugin>3.5.0</version.bundle.plugin>
+        <version.checkstyle.plugin>2.17</version.checkstyle.plugin>
+        <version.clean.plugin>3.0.0</version.clean.plugin>
+        <version.compiler.plugin>3.7.0</version.compiler.plugin>
+        <version.copy.plugin>0.0.6</version.copy.plugin>
+        <version.dependency.plugin>3.0.2</version.dependency.plugin>
+        <version.depends.plugin>1.4.0</version.depends.plugin>
+        <version.deploy.plugin>2.8.2</version.deploy.plugin>
+        <version.enforcer.plugin>3.0.0-M1</version.enforcer.plugin>
+        <version.forbiddenapis.plugin>2.5</version.forbiddenapis.plugin>
+        <version.gmavenplus.plugin>1.6</version.gmavenplus.plugin>
+        <version.install.plugin>2.5.2</version.install.plugin>
+        <version.japicmp.plugin>0.11.0</version.japicmp.plugin>
+        <version.jar.plugin>3.0.2</version.jar.plugin>
+        <version.jqassistant.plugin>1.3.0</version.jqassistant.plugin>
+        <version.javadoc.plugin>3.0.0</version.javadoc.plugin>
+        <version.license.plugin>3.0</version.license.plugin>
+        <version.release.plugin>2.5.3</version.release.plugin>
+        <version.resources.plugin>3.0.2</version.resources.plugin>
+        <version.shade.plugin>3.1.0</version.shade.plugin>
+        <version.sigtest.plugin>1.0</version.sigtest.plugin>
+        <version.source.plugin>3.0.1</version.source.plugin>
+        <version.surefire.plugin>2.21.0</version.surefire.plugin>
+        <version.failsafe.plugin>${version.surefire.plugin}</version.failsafe.plugin>
+        <version.wildfly.plugin>1.2.1.Final</version.wildfly.plugin>
+        <version.org.wildfly.core>5.0.0.Final</version.org.wildfly.core>
+        <version.wildfly-patch-gen.plugin>2.0.1.Alpha5</version.wildfly-patch-gen.plugin>
+        <version.wildfly-patch-gen.plugin.woodstox>5.0.3</version.wildfly-patch-gen.plugin.woodstox>
 
-        <!-- URLs used in javadoc and documentation generation -->
-        <bv.spec.url>http://beanvalidation.org/2.0/spec/</bv.spec.url>
-        <java.api-docs.base-url>http://docs.oracle.com/javase/8/docs/api</java.api-docs.base-url>
-        <java.technotes.base-url>http://docs.oracle.com/javase/8/docs/technotes</java.technotes.base-url>
-        <javaee.api-docs.base-url>http://docs.oracle.com/javaee/7/api</javaee.api-docs.base-url>
-        <javafx.docs.url>http://docs.oracle.com/javase/8/javase-clienttechnologies.htm</javafx.docs.url>
-        <bv.api-docs.base-url>http://docs.jboss.org/hibernate/beanvalidation/spec/2.0/api</bv.api-docs.base-url>
-        <javamoney.api-docs.base-url>http://javamoney.github.io/apidocs</javamoney.api-docs.base-url>
+        <!-- JVM options required for Arquillian and WildFly -->
+        <arquillian.jvm.args></arquillian.jvm.args>
```

Comment:
Note that Arquillian can be used with much more than WildFly, so it may be wise to at least put the term "wildfly" in there if you intend to put wildfly-specific JVM options.

Reply:
Yeah, I had an hesitation here. Looks like you had the same so let's add `wildfly`.